SingularityExecutor/src/main/resources/logrotate.conf.hbs

```diff
@@ -16,6 +16,7 @@ notifempty
 {{#if extrasFiles}}
 {{#each extrasFiles}}{{{this}}} {{/each}} {
   dateformat {{{ extrasDateformat }}}
+  olddir {{{ rotateDirectory }}}
```

Comment:
this is unnecessary, and broke logrotate in hs_staging -- turns out for these, the current directory is already the `rotateDirectory`, so setting this makes it attempt .../logs/logs